application/src/main/java/org/togetherjava/tjbot/commands/free/FreeChannelMonitor.java

```diff
@@ -105,6 +130,18 @@ public boolean isChannelBusy(final long channelId) {
         return requiresIsMonitored(channelId).isBusy();
     }
 
+    /**
+     * This method tests if channel status to busy, see {@link ChannelStatus#isBusy()} for details.
```

Comment:
"tests if channel status **to** busy" is incorrect english. probably meant "is busy".